tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -384,6 +388,10 @@ internal TracerSettings(IConfigurationSource? source, IConfigurationTelemetry te
             StatsComputationEnabledInternal = config
                                      .WithKeys(ConfigurationKeys.StatsComputationEnabled)
                                      .AsBool(defaultValue: (IsRunningInGCPFunctions || IsRunningInAzureFunctionsConsumptionPlan));
+            if (ExperimentalAppsecStandaloneEnabledInternal)
+            {
+                StatsComputationEnabledInternal = false;
```

Comment:
We may want to manually send a configuration telemetry entry here that shows that the setting was overridden. Or, instead of this `if`, maybe you can add `ExperimentalAppsecStandaloneEnabledInternal` to the conditions in `AsBool()` above.

I'm not sure if we want to consider this as "the default value of `StatsComputationEnabledInternal` changes according to `ExperimentalAppsecStandaloneEnabledInternal`" or "the original value was overridden". Do we care about the difference? cc @andrewlock

Reply:
https://github.com/DataDog/dd-trace-dotnet/pull/5565/files/72120d1fb9ce61afb49361747c6888665a04293e#diff-8afc30ae77b35082c438827c3ab04d7dce486226317aeef23e40e7bd62ba2b75